aQute.libg/src/aQute/lib/io/IO.java

```diff
@@ -862,7 +863,14 @@ public static File mkdirs(File dir) throws IOException {
 
 	public static Path mkdirs(Path dir) throws IOException {
 		if (Files.isSymbolicLink(dir)) {
-			return mkdirs(Files.readSymbolicLink(dir));
+			Path target = Files.readSymbolicLink(dir);
+			boolean recreateSymlink = isWindows() && !Files.exists(target, LinkOption.NOFOLLOW_LINKS);
```

Comment:
This could use a comment I think

Reply:
I think it is handled in detail in the commit message, as well as a comment below. Using git blame, you can easily find the commit which modified this line and its commit message. I find that is often more powerful and useful the source code comments. YMMV. :-)

Reply:
Cool. Thanks for nothing 👍